lightning/src/ln/chan_utils.rs

```diff
@@ -858,6 +858,210 @@ pub fn build_anchor_input_witness(funding_key: &PublicKey, funding_sig: &Signatu
 	ret
 }
 
+pub(crate) struct ChannelFundingTransaction {
+	tx: Option<Transaction>,
+	txo: Option<chain::transaction::OutPoint>,
+	/// The hash of the block in which the funding transaction was included.
+	tx_confirmed_in: Option<BlockHash>,
+	tx_confirmation_height: Option<u32>,
```

Comment:
Related though nothing that is being actively worked on our decided: https://github.com/lightningdevkit/rust-lightning/pull/2685#discussion_r1394336586